app/assets/styles/main.css

```diff
@@ -71,55 +71,106 @@ header h1 {
   margin-top: 20px;
 }
 
+.content-holder{
+  display: flex;
+  flex-wrap: wrap;
+  height: 100vh;
+  width: 90%;
+  margin-left: auto;
+  margin-right: auto;
+  justify-content: space-around;
+  margin-top: 45px;
+}
+
 .image-holder{
   display: inline-block;
-  min-height: 10%;
-  max-height: 50%;
-  width: 50%;
-  min-width: 50%;
+  position: relative;
+  width: 45%;
+  min-height: 30%;
+  height: auto;
+  max-height: 55%;
   background-color: #ddd;
   box-shadow: 0px 0px 4px #222;
-  margin-right: 10px;
 }
 
-.content-holder{
-  display: flex;
-  height: 100vh;
-  width: 90%;
+#image{
+  display: block;
+  margin: 0;
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+  max-width: 85%;
+  max-height: 85%;
+  box-shadow: 0px 0px 2px #444;
+}
+
+.car-facts{
+  display: block;
+  width: 95%;
+  height: 40%;
+  margin-top: 20px;
+  background-color: #ddd;
+  box-shadow: 0px 0px 4px #222;
   margin-left: auto;
   margin-right: auto;
-  justify-content: space-between;
-  margin-top: 45px;
 }
 
-.components-holder{
-  height: 100vh;
+.car-facts h2{
+  font-size: 24px;
+}
+
+.details{
+  font-size: 18px;
+  margin-left: 10px;
+  margin-right: 10px;
 }
 
 .results-holder{
   display: inline-block;
-  min-height: 10%;
-  max-height: 50%;
+  min-height: 30%;
   height: auto;
-  width: 50%;
-  min-width: 50%;
+  max-height: 55%;
+  width: 45%;
   background-color: #ddd;
   box-shadow: 0px 0px 4px #222;
-  margin-left: 10px;
+}
+
+.components-holder{
+  height: 100vh;
 }
 
 .results-holder h3{
+  text-align: center;
   font-size: 18px;
   margin-left: 5px;
   margin-right: 5px;
 }
 
 .result{
+  margin-left: auto;
+  margin-right: auto;
+  width: 70%;
+  height: 40px;
+  background-color: #FED750;
+  padding-top: 2px;
+  padding-bottom: 2px;
   margin-top: 20px;
   margin-bottom: 20px;
```

Comment:
You can write a lot of these margin/padding styles in one line. In this case, margin: 20 auto; padding: 2 auto;